File: JAVA/gestionApp/personas/Persona.java
package gestionApp.personas;

public abstract class Persona {
    protected String nombre;
    protected int cedula;
    protected String cel;
    protected String correo;

    public Persona(String nombre, int cedula, String cel, String correo) {
        this.nombre = nombre;
        this.cedula = cedula;
        this.cel = cel;
        this.correo = correo;
    }

    protected abstract String getNombre();
    protected abstract void setNombre(String nombre);

    protected abstract int getCedula();
    protected abstract void setCedula(int cedula);

    protected abstract String getCel();
    protected abstract void setCel(String cel);

    protected abstract String getCorreo();
    protected abstract void setCorreo(String correo);

}

File: Python/gestionApp/personas/Persona.py
class Persona:
    
    def __init__(self,nombre,cedula,cel,correo):
        self.nombre = nombre
        self.cedula = cedula 
        self.cel = cel
        self.correo = correo
    

    def getNombre(self): return self.nombre
    def setNombre(self,nombre): self.nombre = nombre
    
    def getCedula(self): return self.cedula
    def setCedula(self,cedula): self.cedula = cedula
     
    def getCel(self): return self.cel
    def setCel(self,cel): self.cel = cel    
    
    def getCorreo(self): return self.correo
    def setCorreo(self,correo): self.correo = correo   
    
    def toStringDatos(self):
        return "Nombre: " + self.nombre + "\nCedula " + self.cedula + "\nCel: " + self.cel + "\nCorreo: " + self.correo